packages/schematics/angular/module/index_spec.ts

```diff
@@ -42,6 +42,22 @@ describe('Module Schematic', () => {
     expect(files.indexOf('/src/app/foo/foo.module.ts')).toBeGreaterThanOrEqual(0);
   });
 
+  it('should import a generated flat module with a correct relative path', () => {
+    const options = {
+      ...defaultOptions,
+      appRoot: 'app',
+      module: 'app',
+      flat: true,
+    };
+
+    const tree = schematicRunner.runSchematic('module', options, appTree);
+    const files = tree.files;
+    expect(files.indexOf('/src/app/app.module.ts')).toBeGreaterThanOrEqual(0);
+    expect(files.indexOf('/src/app/foo.module.ts')).toBeGreaterThanOrEqual(0);
+    const moduleContent = getFileContent(tree, '/src/app/app.module.ts');
+    expect(moduleContent).toMatch(/import { FooModule } from '.\/foo.module'/);
```

Comment:
Please update regex to this:
```
/import { FooModule } from '\.\/foo.module'/
```
This way it checks for a period and not any character.

Reply:
Absolutely.. added and squashed the requested change